lib/spack/spack/cmd/checksum.py

```diff
@@ -27,12 +27,20 @@ def setup_parser(subparser):
         '--keep-stage', action='store_true',
         help="don't clean up staging area when command completes")
     arguments.add_common_arguments(subparser, ['package'])
+    subparser.add_argument(
+        '-b', '--batch', action='store_true',
+        help="don't ask which versions to checksum")
```

Comment:
Can you move this above the `add_common_arguments` line? That will separate the flags and positional arguments in the code.